person.rb

```diff
@@ -25,4 +26,8 @@ def of_age?
   def can_use_services?
     true if of_age? || @parent_permission
   end
+
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
```

Comment:
- [ ] Hey there! 🙌 I noticed that you've marked the "add person rental" method as private. 🤔 However, it appears that this method should be made public, not private! 📢 Let's make the adjustment to ensure it's accessible to everyone who needs it. 🚀